environs/config/config.go

```diff
@@ -952,6 +972,14 @@ func (c *Config) MaxStatusHistorySizeMB() uint {
 	return uint(val)
 }
 
+// UpdateStatusHookInterval is how often to run the charm
+// update-status hook.
+func (c *Config) UpdateStatusHookInterval() time.Duration {
+	// Value has already been validated.
+	val, _ := time.ParseDuration(c.mustString(UpdateStatusHookInterval))
```

Comment:
The value is a time duration - it's a string eg "30m". There's no ambiguity with units.